### ISSUE TITLE
Tusimple test results

### ISSUE THREAD

**Issue by huangsheng233** [lucastabelini/LaneATT#52]
Thanks for your contribution！I got a problem when testing your provided model.
Is it normal to get a result as the following?
{"Accuracy": 0.9499680129, "FP": 0.0973520249, "FN": 0.0362748622, "FPS": 1000.0}
the command is:
`python main.py test --exp_name laneatt_r122_tusimple`
and my environment is python3.8+cuda10.2+pytorch1.8.
In my opinion, the results of the experiment should not change, since I used the provided model.

**Comment by lucastabelini** [comment]
As written in the README.md, one of the requirements is:
> PyTorch == 1.6, tested on CUDA 10.2. The models were trained and evaluated on PyTorch 1.6. When testing with other versions, the results (metrics) are slightly different.

**Comment by huangsheng233** [comment]
Got it!
After I train a new model, I get a result close to the paper. Thanks!

**Comment by parthdave1210** [comment]
Hi, I have the same environment as you set up on my system. python3.8+cuda10.2+pytorch1.8.
But when I run the evaluation 

python main.py test --exp_name laneatt_r122_tusimple

I get this error can you help me?
![lcr1](https://user-images.githubusercontent.com/32506322/118758668-76c3ee80-b83d-11eb-8a6f-aea28e4a3b04.PNG)
